app/routes.py

```diff
@@ -18,6 +18,7 @@
 
 # Initialize OpenAI API key
 openai_api_key = os.getenv("OPENAI_API_KEY")
+use_openai_headline = os.getenv("USE_OPENAI_HEADLINE", "False").lower() == "true"
```

Comment:
This should check for `== "True"`. This is also being checked at the module level, so make sure that the tests account for that.

Reply:
Thank you for the feedback, @lgaleana. I'll update the check to `== "True"` and ensure the tests account for the module-level evaluation. I'll push the changes shortly.